File: leetcode/algorithms/521_longest_uncommon_subsequence_1/main.js
/**
 * @param {string} a
 * @param {string} b
 * @return {number}
 */
var findLUSlength = function (a, b) {
  if (a === b) {
    return -1;
  } else {
    return Math.max(a.length, b.length);
  }
};


// Solution
var solution = function (a, b) {
  return a === b ? -1 : Math.max(a.length, b.length);
};

File: leetcode/algorithms/521_longest_uncommon_subsequence_1/main.java
class LongestUncommonSubsequenceI {
    public int findLUSlength(String a, String b) {
        return a.equals(b) ? -1 : Math.max(a.length(), b.length());
    }


    // Solution
    public int solution(String a, String b) {
        if (a.equals(b)) {
            return -1;
        }
        return Math.max(a.length(), b.length());
    }
}
